demo/procedures/local_screen_example.rb

```diff
@@ -0,0 +1,51 @@
+temp = 0
+screen = local_screen('Title') do
+'
+  SCREEN AUTO AUTO 0.1 FIXED
+
+  VERTICAL
+    TITLE "Local Variable and Target Display"
+
+    VERTICALBOX
+      LABELVALUE LOCAL LOCAL temp
```

Comment:
LOCAL LOCAL is just the tgt/pkt convention to access the local variable. I tried this with simply VALUE and FORMATVALUE and TEXTBOX with a text variable and all seemed to work well.